src/helpers/routes.js

```diff
@@ -19,11 +19,12 @@ import reducers from '../reducers/reducers';
 import Styles from '../styles/App';
 
 export const Routes = () => {
-  const store = createStore(reducers, applyMiddleware(thunk));
-  sessionService.initSessionService(store);
-  store.subscribe(() => {
-    console.log('store.getState()', store.getState());
+  const Store = createStore(reducers, applyMiddleware(thunk));
+  sessionService.initSessionService(Store);
+  Store.subscribe(() => {
+    console.log('store.getState()', Store.getState());
   });
+  const [store] = useState(Store);
```

Comment:
1) I don't think you need to assign it to a variable. I think throwing into state is good enough
2) Make `Store` lowercase

Reply:
I can't have both be called store otherwhere it acts as having two variables named store. So what should i do?

Reply:
Exactly. You don't need to assign the result of `useState`. This is only to put it into the state. We would assign it if we needed access afterwards.

Reply:
but even initializing const[store]; still sees two store variables so we can't have them be named the same thing.

Reply:
We don't need the lowercase store. There's no point to it. If you get rid of it, you can make the uppercase one lowercase.

Reply:
done